src/Controllers/GagePageController.ts

```diff
@@ -536,13 +581,300 @@ module StreamStats.Controllers {
+                });
+            }
+            if (this.estPeakDates) {
+                this.formattedEstPeakDates = [];
+                this.estPeakDates.forEach(test => {
+                    this.formattedEstPeakDates.push({x: new Date(test.peak_dt), y: test.peak_va})
+                });
+            }
+            if (this.dailyFlow) {
+                this.formattedDailyFlow = [];
+                this.dailyFlow.forEach(test => {
+                    if (test.qualifiers[0] === 'A') {
+                    this.formattedDailyFlow.push({x: new Date(test.dateTime), y: parseInt(test.value)})
+                    }
+                });
+            }
+            if (this.floodFreq) { //set up AEP plotLines
+                this.formattedFloodFreq = [];
+                    const AEPColors = {
+                        9: '#9A6324',
+                        852: '#800000',
+                        8: '#e6194B',
+                        818: '#ffd8b1',
+                        7: '#f58231',
+                        6: '#ffe119', 
+                        5: '#bfef45',
+                        4: '#3cb44b',
+                        3: '#42d4f4',
+                        1: '#4363d8',
+                        501: '#000075',
+                        2: '#911eb4',
+                        500: '#dcbeff',
+                        851: '#fabed4',
+                        1438: '#469990'
+                    };
+                this.floodFreq.forEach((floodFreqItem) => {
+                    let colorIndex = floodFreqItem.regressionTypeID;
+                    let formattedName = floodFreqItem.regressionType.name.substring(0, floodFreqItem.regressionType.name.length-18);
+                    this.formattedFloodFreq.push({
+                        value: floodFreqItem.value,
+                        color: AEPColors[colorIndex],
+                        width: 1.5,
+                        zIndex: 4,
+                        label: {text: formattedName + '% AEP'}
+                        });
+                    });
+            this.createAnnualFlowPlot();
+        }}
+
+        //Create chart
+        public createAnnualFlowPlot(): void {
+            console.log('peak value plot data', this.formattedPeakDates);
+            console.log('estimated peak plot data', this.formattedEstPeakDates);
+            console.log('daily flow plot data', this.formattedDailyFlow);
+            this.chartConfig = {
+                chart: {
+                    height: 450,
+                    width: 800,
+                    zooming: {
+                        type: 'xy'
+                    }
+                },
+                title: {
+                    text: 'Annual Peak Streamflow',
+                    align: 'center'
+                },
+                subtitle: {
+                    text: 'Click and drag in the plot area to zoom in<br>AEP: Annual Exceedance Probability',
```

Comment:
Good idea to describe AEP. Let's say "AEP = Annual Exceedance Probability" instead of the colon.